docs/src/explanations/chisel-enum.md

```diff
@@ -13,9 +13,9 @@ In contrast with `Chisel.util.Enum`, `ChiselEnum` are subclasses of `Data`, whic
 
 ```scala mdoc
 // Imports used in the following examples
+import circt.stage.ChiselStage
```

Comment:
any particular reason you moved this line?

Reply:
To get rid of `_root_.` per Jack's fine suggestion.

Reply:
But perhaps this one just got caught up in the refactoring.

Reply:
I thought it was required because the build was failing.  Figuring out how to build the doc examples locally.

Reply:
no, it's needed too.

Reply:
😂

Reply:
yah, well I guess it is good if we are consistent the order.